instrumentation-genai/opentelemetry-instrumentation-openai-v2/examples/zero-code/README.rst

```diff
@@ -1,4 +1,4 @@
-OpenTelemetry OpenAI Instrumentation Example
+OpenTelemetry OpenAI Zero-Code Instrumentation Example
 ============================================
```

Comment:
Another nit about the zero-code example. `opentelemetry-instrument` will use otlp and grpc by default, which differs slightly from the coded example that uses otlp/http.

If you want to modify the zero-code example to use EXACTLY the same components:

`dotenv run -- opentelemetry-instrument --traces_exporter otlp_proto_http --metrics_exporter otlp_proto_http --logs_exporter otlp_proto_http python main.py`

Reply:
It uses http/protobuf (see .env file),  but I updated it to use grpc. 

I don't think we need exactly the same set of components. We'll also add metrics to openai instr and will update manual example to setup metrics pipeline too.

BTW - https://github.com/open-telemetry/opentelemetry-python-contrib/pull/3042 - we don't default to otlp exporter for logs 🙃